src/Containers/CRMTable/CRMTable.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+    Column,
+    useFilters,
+    useTable
+} from 'react-table';
+import { CRMRowProps } from '../CRMRow/CRMRow';
+
+
+export interface ICRMTableProps extends React.HTMLAttributes<HTMLTableElement> {
+    data: Array<CRMRowProps>; 
+    columns: Array<Column<CRMRowProps>>;
+    defaultColumn?: any,
```

Comment:
Don't use the any type

Reply:
The issue I've had with the defaultColumn is that its an object with two fields at least and I can't seem to find any prebuilt type in react-table that can be used for it like I had for Column. So should I create my own class to represent the fields inside of the object or what. Because for filtering the default options you need to give for every column is "Filter" which is the component to render for the filter and "filter" which is the function used for filtering each with their own types.

Though I might also be able to move away from having default columns in general but in that case, every column has to declare if its filtered or not.

Reply:
Alright it should be properly typed now